### PR TITLE
reference: unify notation for targets

### PR DESCRIPTION
**Pull request by sago35**
see #226

### CODE REVIEW

**Comment by sago35** [comment]
This is the same notation and order as tinygo-org/tinygo/README.md, which makes it easier to check for missing documentation.
This allows, for example, STM32 microcontrollers to be lined up close together.

image

File: content/docs/reference/microcontrollers/hifive1b.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "HiFive1 RevB"
+title: "SiFive HiFive1"
```
Comment (aykevl) [inline content/docs/reference/microcontrollers/hifive1b.md]: The HiFive1 and HiFive1 Rev B are slightly different (incompatible) boards.

```suggestion
title: "SiFive HiFive1 Rev B"
```

**Review by aykevl — COMMENTED** [review]
Looks reasonable to me, but see the comment below.

**Comment by sago35** [comment]
Updated.

After this, create a PR that also modifies tinygo-org/tinygo/README.md to `SiFive HiFive1 Rev B` .

**Comment by sago35** [comment]
@aykevl 
Please check it.

**Comment by deadprogram** [comment]
Good cleanup here @sago35 thanks! Now merging.

**Pull request merged**